### ISSUE TITLE
Univ of Wyoming dept seminar and RLadies - Oct 7

### ISSUE THREAD

**Issue by jules32** [Openscapes/how_we_work#365]

**Comment by jules32** [comment]
Molly Caldwell organizing.

Schedule:

faculty members here volunteered to help facilitate the R Ladies round table on open data science ([Dr. Grace Shearrer](https://www.m2aenad.app/home/about) and[ Dr. Katie Wagner](https://cewagnerlab.com/41-2/)), so we should have a good variety of perspectives.
 ![Image](https://github.com/user-attachments/assets/c15851a7-2271-4b01-bbf6-34b7308ef160)

**Comment by jules32** [comment]
This went nicely! Was great to meet Molly Caldwell, INBRE collaborators, and other RLadies and scientists and open science folks throughout the day. 

Slides on Media page. 

Bio: Julia Stewart Lowndes, PhD
Openscapes core team member and founding director

I am a marine ecologist working at the intersection of actionable science, data science, and open science. My main focus is mentoring teams to develop technical and leadership mindsets and skills for data-intensive research, grounded in climate solutions, inclusion, and kindness. I earned my PhD from Stanford University in 2012 studying drivers and impacts of Humboldt squid in a changing climate. I am an active participant in the open science community, and co-founded RLadies Santa Barbara. In 2018 I founded Openscapes as an open source community following my own Ocean Health Index research team’s path to better science in less time, as a Mozilla Fellow and Senior Fellow at the National Center for Ecological Analysis and Synthesis (NCEAS) at the University of California Santa Barbara. In 2022 I started Openscapes LLC, a woman-owned small business to support the growing Openscapes open source community.
